backend/de.metas.salescandidate.base/src/main/java/de/metas/ordercandidate/api/OLCandOrderFactory.java

```diff
@@ -315,6 +321,17 @@ private I_C_Order newOrder(@NonNull final OLCand candidateOfGroup)
 		order.setEMail(candidateOfGroup.getEmail());
 		order.setPhone(candidateOfGroup.getPhone());
 		order.setM_SectionCode_ID(SectionCodeId.toRepoId(getSectionCodeId(candidateOfGroup)));
+		final AuctionId auctionId = candidateOfGroup.getAuctionId();
+		if (auctionId != null)
+		{
+			order.setC_Auction_ID(AuctionId.toRepoId(auctionId));
```

Comment:
imho u shall move this inside `if(action!=null)`

Reply:
done.